docs/operations/operations.md

```diff
@@ -228,13 +228,15 @@ After the service principal secret has been rotated and the corresponding secret
 
 ### Rolling Update Triggers
 
-Changes to the `Shoot` worker-pools are applied in-place where possible. In case this is not possible a rolling update of the workers will be performed to apply the new configuration, as outlined in [the Gardener documentation](https://github.com/gardener/gardener/blob/master/docs/usage/shoot-operations/shoot_updates.md#in-place-vs-rolling-updates). The exact fields that trigger this behaviour depend on whether the feature gate `NewWorkerPoolHash` is enabled. If it is not enabled, the fields mentioned in the [Gardener doc](https://github.com/gardener/gardener/blob/master/docs/usage/shoot-operations/shoot_updates.md#rolling-update-triggers) are used, with a few additions:
+Changes to the `Shoot` worker-pools are applied in-place where possible.
+In case this is not possible a rolling update of the workers will be performed to apply the new configuration, 
+as outlined in [the Gardener documentation](https://github.com/gardener/gardener/blob/master/docs/usage/shoot-operations/shoot_updates.md#in-place-vs-rolling-updates).
+The exact fields that trigger this behavior are defined in the [Gardener doc](https://github.com/gardener/gardener/blob/master/docs/usage/shoot-operations/shoot_updates.md#rolling-update-triggers), 
+with a few additions:
 
 - `.spec.provider.infrastructureConfig.identity`
 - `.spec.provider.workers[].dataVolumes[].size` (only the affected worker pool)
 - `.spec.provider.workers[].dataVolumes[].type` (only the affected worker pool)
 
-If the feature gate _is_ enabled, instead of the complete provider config only the fields explicitly mentioned above are used, with the addition of
-
-- `.spec.provider.workers[].providerConfig.diagnosticsProfile.enabled`
-- `.spec.provider.workers[].providerConfig.diagnosticsProfile.storageURI`
+For now, if the feature gate `NewWorkerPoolHash` _is_ enabled, the exact same fields are used.
```

Comment:
as with the other PRs please add the providerConfig and also the changes from the cloudprofile with regards to the faultdomains and VMSS